app/controllers/catalog_manager/identities_controller.rb

```diff
@@ -32,7 +32,7 @@ def associate_with_org_unit
     identity = Identity.find identity_id
 
     if rel_type == 'service_provider_organizational_unit'
-      if not oe.service_providers or (oe.service_providers and not oe.service_providers.map(&:id).include? identity_id)
+      if not oe.service_providers or (oe.service_providers and not oe.service_providers.map(&:identity_id).include? identity_id)
```

Comment:
This is minor, but for efficiency, `pluck(:identity_id)` would be much more efficient since `#map` requires the objects to be returned from the database

Reply:
Yes. I changed map to pluck. and committed the changes